libheif/bitstream.cc

```diff
@@ -268,6 +268,14 @@ uint32_t BitstreamRange::read32()
                      (buf[3]));
 }
 
+float BitstreamRange::readFloat32()
+{
+    int i = read32();
```

Comment:
There is a `BitstreamRange::read32_float()` function in the `timestamp-properties` branch. It didn't get merged (yet) and the TAI timestamps changed their use of `float` to `int`. It seems that the two implementations are equivalent.

Could we use a `reinterpret_cast` instead of the `memcpy`? And `uint32_t` instead of `int`.

I checked how endinanness works for floating point. It seems that all modern architectures use the same endianness for floats as for integers. The only exception seems to be some ARM processors that use a mixed-endian format for 64bit floats.